index.html

```diff
@@ -207,17 +207,21 @@ <h4>Skills</h4>
                         </ul>
                     </div>
                 </li>
-            </ul>
-            
+            </ul>        
+        </section>
+
+        <section id="contact-form">
             <div class="form-info">
                 <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
             </div>
 
-            <input type="text" class="input" placeholder="Full name">
-            <input type="email" class="input" placeholder="Email adress">
-            <textarea name="" id="" cols="30" rows="10" class="input textarea" placeholder="Enter text here"></textarea>
-            
-            <a href="#" class="button">Get in touch</a>
+            <form action="https://formspree.io/f/xgebvyqg" method="post">
+                <input type="text" name="name" class="input" placeholder="Full name" maxlength="30" required>
+                <input type="email" name="email" class="input" placeholder="Email adress" required>
+                <textarea name="message" id="" cols="30" rows="10" class="input textarea" placeholder="Enter text here" maxlength="500" required></textarea>
+                <input type="submit" id="submit-button" class="button" value="Get in touch">
```

Comment:
👏 I love your creativity on these inputs when they become active, as well as your button states... 💯 Good job, @NicolasAndrehh and @KenbonTN 😄